src/components/Switchers/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledInput = styled.input`
+  &:checked + span {
+    transform: translateX(100%);
+  }
+`
+
+const StaticSwitcher = () => (
+  <div className='w5 relative flex items-center br-pill ba b--light-navy bg-dark-navy'>
+    <label className='w-50 normal lh-copy tc light-gray relative z-1 pv2 pointer'
+           htmlFor='some-id-left'
+           style={{boxSizing: 'border-box'}}>Monthly</label>
```

Comment:
It works, but as applied class. Idea is to pt it in some stylesheet and use across all styles as default